src/reactivity/unwrap.ts

```diff
@@ -27,6 +27,11 @@ export function unwrapRefProxy(value: any, map = new WeakMap()) {
     (s) => (obj[s] = (value as any)[s])
   )
 
+  // copy __ob__
+  if (value.__ob__) {
```

Comment:
shouldn't that be part of the `Object.keys`? if so can we just add it there when the key is `__ob__`?

Reply:
[It's defined by `defineProperty`](https://github.com/vuejs/vue/blob/0b31647c41b0c916d40b962723b1de06672d0e01/src/core/observer/index.js#L46) which is not enumerable by `Object.keys`